portafly/src/components/data-list/modals/ChangeStateModal.tsx

```diff
@@ -39,40 +37,37 @@ const ChangeStateModal: React.FunctionComponent<Props> = ({
         addAlert({ id: 'success', variant: 'success', title: t('toasts.change_state_start') })
       })
       .catch(() => {
-        actionFailed()
-        addAlert({ id: 'error', variant: 'danger', title: t('toasts.change_state_error') })
+        const error = t('toasts.change_state_error')
+        actionFailed(error)
```

Comment:
Do we mean to always hide the actual error and not even log it?

Reply:
Hide it how? It's shown in the UI. On the other hand the real error will come from then API, this is really yet another placeholder.

Reply:
I meant the raised error